ytmusicapi/setup.py

```diff
@@ -25,7 +25,10 @@ def setup(filepath: str = None, headers_raw: str = None) -> Dict:
 def setup_oauth(filepath: str = None,
                 session: requests.Session = None,
                 proxies: dict = None,
-                open_browser: bool = False) -> Dict:
+                open_browser: bool = False,
+                env_credentials: bool = False,
```

Comment:
Please remove this, as stated I am opposed to the introduction of environment variables. This is not a concern for this library